applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -552,7 +552,8 @@ void enter_dfumode(void)
 	if (slm_operation_mode != SLM_DFU_MODE) {
 		slm_operation_mode = SLM_DFU_MODE;
 		/* only UART Send */
-		uart_receive_disable(true);
+		uart_receive_disable();
+		k_sleep(K_MSEC(10));
```

Comment:
Delay time can be added in uart_receive_disable(), for CMSIS register update

Reply:
Based on tests failing in CI and @junqingzou:s comments. Changing approach.